server/src/database.js

```diff
@@ -0,0 +1,139 @@
+const Sequelize = require('sequelize');
+
+const createStore = () => {
+  const sequelize = new Sequelize({
+    dialect: 'sqlite',
+    storage: './database.sqlite',
+  });
+
+  sequelize.authenticate().then(() => {
+    console.log('Connection to the database has been established successfully');
+  }).catch(err => {
+    console.error(`Unable to connect to the database: ${err}`);
+  });
+
+  const Model = Sequelize.Model;
+
+  class User extends Model {
+  }
+
+  class Event extends Model {
+  }
+
+  User.init({
+        id: {
+          type: Sequelize.INTEGER,
+          primaryKey: true,
+          autoIncrement: true,
+        },
+        createdAt: Sequelize.DATE,
+        firstName: {type: Sequelize.STRING, allowNull: false},
+        lastName: Sequelize.STRING,
+        email: {
+          type: Sequelize.STRING, allowNull: false,
+        },
+        token: Sequelize.STRING,
+        birthday: Sequelize.DATE,
+        city: Sequelize.STRING,
+        state: Sequelize.STRING,
+        phone: Sequelize.STRING,
+
+      },
+      {
+        sequelize,
+        modelName: 'user',
+      });
+
+  class Tag extends Model {
+  }
+
+  Event.init({
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    createdAt: Sequelize.DATE,
+    updatedAt: Sequelize.DATE,
+    hostId: {
+      type: Sequelize.INTEGER,
+      references: {
+        model: User,
+        key: 'id',
+      },
+    },
+    title: Sequelize.STRING,
+    description: Sequelize.STRING,
+    price: Sequelize.FLOAT,
+    maxParticipants: Sequelize.INTEGER,
+  }, {
+    sequelize,
+    modelName: 'event',
+  });
+  Tag.init({
```

Comment:
Not exactly sure what this is - can you add a comment explaining what Tag is and why we need it?

Reply:
We designed tag as a sort of category. Every event can have multiple tags. Tags are predefined by ours(developers) and used by the event host to categorize his event. So we are able to categorize events by tags.

Reply:
When I ask a question during the review, it's because it is like likely that other people will ask the same question later on. I'm not expecting you to explain your justification under my review - it'd be better if you can add your explanation in the code as a comment (or commend block) so other people can read and understand later.

Reply:
Oh... I thought you want me to explain it in this comment window. Sorry for the misunderstanding. I would add a comment in the code.